core/src/main/resources/lib/layout/isAdminOrManage.jelly

```diff
@@ -0,0 +1,6 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:d="jelly:define" xmlns:s="jelly:stapler">
+  <j:if test="${h.hasPermission(app.ADMINISTER) || h.hasPermission(app.MANAGE)}">
```

Comment:
Naming-wise, it seems to make more sense for this to be called `isAdminOrManager`. Or perhaps `hasAdmin[ister]OrManage`?